examples/resiliency/inprocess_restart.py

```diff
@@ -0,0 +1,272 @@
+#!/usr/bin/env python3
```

Comment:
Are we testing this example?

Reply:
no it's not run through CI, similar to https://github.com/NVIDIA-NeMo/Megatron-Bridge/tree/main/examples/recipes/llama